packages/remirror__extension-callout/src/callout-utils.ts

```diff
@@ -24,12 +24,23 @@ export const dataAttributeEmoji = 'data-callout-emoji';
 export function isValidCalloutAttributes(
   attributes: ProsemirrorAttributes,
 ): attributes is CalloutAttributes {
-  return !!(
-    attributes &&
-    isObject(attributes) &&
-    isString(attributes.type) &&
-    attributes.type.length > 0
-  );
+  if (attributes && isObject(attributes)) {
+    const attributesChecklist = Object.entries(attributes).map(([key, value]) => {
+      switch (key) {
+        case 'type':
+          return !!(isString(value) && value.length > 0);
+
+        case 'emoji':
+          return !!(isString(value) && value.length > 0);
+
+        default:
+          return true;
```

Comment:
@whawker Should I use return true in the default case? 
Which means the Callout extension can take attributes except 'type' and 'emoji' and will not constrain those value forms.

Reply:
Yeah that make sense I think, with the `extraAttributes` functionality Remirror has.